src/lib/utils.tsx

```diff
@@ -69,3 +73,139 @@ export const useProjectId = (props: { projectId?: string }) => {
 
   return projectId;
 };
+
+/**
+ * 
+ * @param data
+ * text data
+ * @param count 
+ * the variable number used to determine which plural case to use
+ * zero = 0
+ * one = 1
+ * two = 2
+ * few = 3,4,5
+ * many = 6,7,...,99
+ * other = 100, 101, ...
+ * if count is provided but not matching plural key, fallback to base plural value
+ * in future we should also user's to define their own middleware for picking plurals
+ */
+const getPluralText = (data: any, count: Count) => {
```

Comment:
I think it's pretty important that we strongly type this and avoid the `any`. Most bugs that have bitten me while working on this project have been related to null checking / edge case handling -- the huge number of possible prop configurations + `source` data types makes it really easy to make false assumptions about the data being passed around.

I'd be happy with an interface with the minimum surface area for the implementation of the function -- in this case maybe this?
```ts
getPluralText = (data: { text: string; plurals?: Plurals }, ...)
```

Reply:
👍